runtime/crab/src/lib.rs

```diff
@@ -47,8 +47,9 @@ pub use wasm::*;
 pub mod messages;
 pub use messages::*;
 
-/// Weights for pallets used in the runtime.
-mod weights;
+// TODO: Benchmark
+// /// Weights for pallets used in the runtime.
+// mod weights;
```

Comment:
Remember to update `pub type UncheckedExtrinsic =
	fp_self_contained::UncheckedExtrinsic<Address, Call, Signature, SignedExtra>;`